Mathlib/Combinatorics/SimpleGraph/Walk.lean

```diff
@@ -1311,10 +1311,30 @@ theorem map_injective_of_injective {f : G →g G'} (hinj : Function.Injective f)
       apply ih
       simpa using h.2
 
+section mapLe
+
+variable {G G' : SimpleGraph V} (h : G ≤ G') {u v : V} (p : G.Walk u v)
+
 /-- The specialization of `SimpleGraph.Walk.map` for mapping walks to supergraphs. -/
-abbrev mapLe {G G' : SimpleGraph V} (h : G ≤ G') {u v : V} (p : G.Walk u v) : G'.Walk u v :=
+abbrev mapLe : G'.Walk u v :=
   p.map (.ofLE h)
 
+lemma length_mapLe_eq_length : (p.mapLe h).length = p.length := by simp
+
+lemma support_mapLe_eq_support : (p.mapLe h).support = p.support := by simp
+
+lemma edges_mapLe_eq_edges : (p.mapLe h).edges = p.edges := by simp
+
+lemma edgeSet_mapLe_eq_edgeSet : (p.mapLe h).edgeSet = p.edgeSet := by simp
```

Comment:
I'm not convinced that any of these are necessary; since `mapLe` is an abbreviation, `rw` can work with the `map` version already, eg `rw [length_map]` proves `length_mapLe_eq_length`. 
I'm happy to add the last two in your PR though.